core/src/Revolution/modConnectorResponse.php

```diff
@@ -101,7 +101,7 @@ public function outputContent(array $options = [])
         $target = preg_replace('/[\.]{2,}/', '', htmlspecialchars($options['action']));
 
         $siteId = $this->modx->user->getUserToken($this->modx->context->get('key'));
-        $isLogin = $target == 'Login' || $target == 'Security/Login';
+        $isLogin = $target == 'Login' || $target == 'Security/Login' || $target == 'security/login';
```

Comment:
It shouldn't be added here because it replacement of old 2.x style actions. Need to fix a place where it used.